hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AbstractFSWAL.java

```diff
@@ -816,6 +816,11 @@ protected void archiveLogFile(final Path p) throws IOException {
         i.preLogArchive(p, newPath);
       }
     }
+    if (!fs.exists(this.walArchiveDir)) {
```

Comment:
Why this directory does not exist?

Reply:
> 

At present, it has not been located in which link to delete it, so make another judgment in archiveLogFile